src/sentry/static/sentry/app/actionCreators/organization.tsx

```diff
@@ -85,7 +86,13 @@ export async function fetchOrganizationDetails(
       promises.push(fetchProjectsAndTeams(slug));
     }
 
-    const [org] = await Promise.all(promises);
+    const [org, projectsAndTeams] = await Promise.all(promises);
```

Comment:
Would another way to solve this problem be to have the OrganizationStore to keep the projects/teams as state and apply that state to the organization when it is loaded, and then remove that state? I'm not suggesting making that change, I'm interested in checking that I understand the problem correctly.

Reply:
That could also work; but that seems a more invasive change on the part of the OrganizationStore.